spaceship/lib/spaceship/connect_api/models/bundle_id_capability.rb

```diff
@@ -59,14 +74,26 @@ def is_type?(type)
       # API
       #
 
-      def self.all(client: nil, filter: {}, includes: nil, limit: nil, sort: nil)
+      def self.all(client: nil, bundle_id_id: nil, filter: {}, includes: nil, limit: nil, sort: nil)
+        raise "bundle_id_id is required " if bundle_id_id.nil?
+
+        client ||= Spaceship::ConnectAPI
+        resp = client.get_bundle_id_capabilities(bundle_id_id: bundle_id_id, filter: filter, includes: includes).all_pages
+        return resp.flat_map(&:to_models)
+      end
+
+      def self.create(client: nil, bundle_id_id: nil, capability_type: nil, settings: [])
+        raise "bundle_id_id is required " if bundle_id_id.nil?
+        raise "capability_type is required " if capability_type.nil?
```

Comment:
```suggestion
      def self.create(client: nil, bundle_id_id:, capability_type:, settings: [])
```

We can now remove the default value of keyword argument since we support Ruby from v2.4😄